<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BTSRMain</class>
 <widget class="QDialog" name="BTSRMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
background-color: rgb(41, 188, 204);
}
QToolButton{
background-color:transparent;
color:white;
font-size:24px;
border:1px solid white;
}</string>
  </property>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>70</y>
     <width>681</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:yellow;
font-size:28px;</string>
   </property>
   <property name="text">
    <string>BTSR设置</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_setid">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>120</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>编号</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoRaise">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_yarn_parameter">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>120</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>纱线参数</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoRaise">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton__study">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>200</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>学习</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoRaise">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_sensor_parameter">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>200</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>感应器参数</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoRaise">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_function_set">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>280</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>功能设定</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoRaise">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_sensor_disable">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>280</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>感应器禁用</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoRaise">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="return_btn">
   <property name="geometry">
    <rect>
     <x>752</x>
     <y>380</y>
     <width>48</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10;background-color: rgb(100, 100, 100);font-size:22px;color:rgb(255, 255, 0);

</string>
   </property>
   <property name="text">
    <string>返
回</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
